docs/_content/ontwikkelaars/algemeen/archiveren.md

```diff
@@ -49,15 +53,100 @@ gerelateerde entiteiten:
 
 ### Wat gebeurt er als een zaak-dossier *blijvend bewaard* wordt?
 
-Het zaak-dossier is overgedragen naar een bewaarplaats.
+Het zaak-dossier wordt overgedragen naar een bewaarplaats.
 
 ### Wat gebeurt er als een zaak-dossier *vernietigd* wordt?
 
 Het vernietigen is het definitief verwijderen van data volgens de NEN2082 (Eis 80):
 
 > Vernietigen van archiefstukken/archiefbestanddelen moet zo gebeuren dat deze niet meer op enigerlei wijze kunnen worden gereproduceerd.
 
-## Gerelateerde attributen
+### Bepalen van de aan archivering gerelateerde attributen voor `ResultaatType`
+
+Zie: [Selectielijst gemeenten en intergemeentelijke organen 2017](https://vng.nl/files/vng/20170706-selectielijst-gemeenten-intergemeentelijke-organen-2017.pdf)
+
+Het ZTC dient ingericht te worden volgens de selectielijst. Het ZTC dient validaties uit te voeren om te zorgen dat inrichting correct is.
+
+*Voorbeeld*
+
+Als `ResultaatType.BrondatumArchiefprocedure.Afleidingswijze` wordt ingesteld op `eigenschap` moet het ZTC valideren dat het betreffende `ZaakType` een `Eigenschap` heeft met als `naam`, de waarde die staat in `ResultaatType.BrondatumArchiefprocedure.datumkenmerk`.
+
+### Berekenen van de `Zaak.Archiefactiedatum`
+
+1. Bepaal de *brondatum* van de `Zaak` door de `ResultaatType.BrondatumArchiefprocedure.Afleidingswijze` te raadplegen:
+
+   Afleidingswijze | Waarde van *brondatum*
+   --- | ---
+   `afgehandeld` | `Zaak.Einddatum`
+   `gerelateerde zaak` | *TODO: Wat is dit precies?* De hoogste datum van van alle `Zaak.GerelateerdeZaak.Einddatum` of `Zaak.Einddatum` ([#776](https://github.com/VNG-Realisatie/gemma-zaken/issues/776))
```

Comment:
En Het 'Wijzigingsvoorstel ImZTC en RGBZ a.g.v. Selectielijst'-document zegt het volgende (p. 11) 

> “gerelateerde zaak”: de archiefactiedatum ligt het aantal jaren van de archiefactietermijn na de einddatum van de gerelateerde zaak (of de te bepalen zaak van één van de gerelateerde zaken).

Dit impliceert, samen met Hugo's comment in #776, de eerste optie: "De hoogste datum van van alle `Zaak.GerelateerdeZaak.Einddatum`s"

Reply:
Het RGBZ heeft echter geen relatie die "GerelateerdeZaak" heet. Dus wat wordt er wel mee bedoeld?